CHANGES.rst

```diff
@@ -10,7 +10,10 @@ New Features
 
 - ``astropy.convolution``
 
-  - Major change in convolution behavior and keyword arguments (#5782)
+  - Major change in convolution behavior and keyword arguments.   Additional
+    details are in the API section. (#5782)
```

Comment:
there are other examples of (#x) in the changelog; maybe fix those in another PR?